src/app/dashboard/page.tsx

```diff
@@ -7,8 +7,15 @@ import {
   CardTitle,
 } from "@/components/ui/card";
 import { StoreForm } from "./store-form";
+import axios from "axios";
 
-export default function Dashboard() {
+export default async function Dashboard() {
+  // TEST TURSO
+  const result = await axios.post("http://localhost:3000/api/dummy/", {
```

Comment:
E' questo il modo giusto per chiamare un'api in next.js

Reply:
Mh ok. Giusto! Però sono comunque indeciso se separare la logica con dei service. E' un pò brutto avere l'SQL spiattellato, e magari ripetuto nei components. Cioè se si deve fare una query da più parti come dici tu la si deve riscrivere N volte. Tuttavia capisco che non serve fetcharla ma si chiama direttamente la funzione